.github/workflows/build_playground_backend.yml

```diff
@@ -0,0 +1,73 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build And Deploy Playground Backend Application
+
+on:
+  push:
+    branches: ['master', 'release-*']
+    tags: 'v*'
+  pull_request:
+    branches: ['master', 'release-*']
+    tags: 'v*'
+    paths: ['playground/backend/**']
+  workflow_dispatch:
+
+jobs:
+  build_playground_backend_docker_image:
+    name: Build Playground Backend App
+    runs-on: ubuntu-latest
+    env:
+      GO_VERSION: 1.17.0
+      BEAM_VERSION: 2.33.0
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: joschi/setup-jdk@v2
+        with:
+            java-version: '8'
+      - uses: actions/setup-go@v2
+        with:
+            go-version: '${{ env.GO_VERSION }}'
+      - name: Prepare Go lint env
+        run: "sudo ./playground/backend/env_setup.sh"
+      - name: Run Lint
+        run: "golangci-lint run internal/..."
+        working-directory: playground/backend/
+        continue-on-error: true
+      - name: Run Tests
+        run: ./gradlew playground:backend:test
+        continue-on-error: true
+      - name: install npm
+        uses: actions/setup-node@v2
+        with:
+            node-version: '14'
+      - name: install docker linter
+        run: npm install  -g dockerlint
+      - name: lint dockerfile
+        run: dockerlint Dockerfile
+        working-directory: playground/backend/containers/java
+      - name: Setup GCP account
+        run: echo ${{ secrets.GCP_ACCESS_KEY }} | base64 -d > /tmp/gcp_access.json
```

Comment:
Are the secrets/passwords going to be visible in the logs?

Reply:
There will be no output generated for that command, because it will be redirected to file, but even if there will be output, it will be masked